codegen/datamodelgen.py

```diff
@@ -306,7 +306,7 @@ def _write_doc_for_model_object(
             ref = "_ref_" + "_".join([x.strip("_") for x in heading.split(".")])
             f.write(f".. {ref}:\n\n")
             if class_name == "Root":
-                heading_ = heading.replace("_", "\_")
+                heading_ = heading
```

Comment:
`.replace("_", "\_")` doesn't do anything currently as none of the `heading` values contain underscores, same for `tuigen.py`, these are all the possible `heading` values (v241 dev version of Oct 23):
```
meshing.tui
solver.tui
meshing.datamodel.workflow
solver.datamodel.workflow
meshing.datamodel.meshing
meshing.datamodel.PartManagement
meshing.datamodel.PMFileManagement
solver.datamodel.flicing
meshing.datamodel.preferences
solver.datamodel.preferences
solver.datamodel.solverworkflow
```